sw/device/silicon_creator/manuf/lib/individualize_sw_cfg.h

```diff
@@ -85,6 +86,21 @@ OT_WARN_UNUSED_RESULT
 status_t manuf_individualize_device_creator_manuf_state_cfg(
     const dif_otp_ctrl_t *otp_ctrl);
 
+/**
+ * Configures the IMMUTABLE_ROM_EXT_EN field in the CREATOR_SW_CFG OTP
+ * partition.
+ *
+ * This must be called before `manuf_individualize_device_creator_sw_cfg_lock()`
+ * is called. The operation will fail if there are any pre-programmed words not
+ * equal to the expected test values.
+ *
+ * @param otp_ctrl OTP controller instance.
+ * @return OK_STATUS if the IMMUTABLE_ROM_EXT_EN field was provisioned.
+ */
+OT_WARN_UNUSED_RESULT
+status_t manuf_individualize_device_immutable_rom_ext_en_cfg(
```

Comment:
Just a thought here, since we now have a separate function for setting the:
- flash_data_default_cfg
- manuf_state
- immutable_rom_ext_en
- rom_bootstrap_dis
perhaps we could combine these into a one function that take as inputs:
- the OTP partition
- the field / offset to program
?

This would consolidate a lot of very similar functionality. Ok to do in a follow up PR, or additional commit on this PR if you would like.

Reply:
Thanks for the suggestion. That's a good improvement. If you don't mind,  I could include this in the PR for #24617